src/Squidex/app/shared/guards/must-be-authenticated.guard.spec.ts

```diff
@@ -9,24 +9,25 @@ import { Router } from '@angular/router';
 import { of } from 'rxjs';
 import { IMock, It, Mock, Times } from 'typemoq';
 
-import { AuthService } from '@app/shared';
+import { AuthService, UIOptions } from '@app/shared';
 
 import { MustBeAuthenticatedGuard } from './must-be-authenticated.guard';
 
 describe('MustBeAuthenticatedGuard', () => {
     let router: IMock<Router>;
-
     let authService: IMock<AuthService>;
-    let authGuard: MustBeAuthenticatedGuard;
+    let uiOptions = new UIOptions({ map: { type: 'OSM' } });
+    let uiOptionsRedirect = new UIOptions({ map: { type: 'OSM' }, redirectToLogin: true });
 
     beforeEach(() => {
         router = Mock.ofType<Router>();
 
         authService = Mock.ofType<AuthService>();
-        authGuard = new MustBeAuthenticatedGuard(authService.object, router.object);
```

Comment:
Just out of interest, how comes this was moved out of the 'BeforeEach' section if it is used in all tests within this file?

Reply:
It is used with different arguments.